python/cudf_polars/cudf_polars/experimental/benchmarks/utils.py

```diff
@@ -214,12 +212,21 @@ def from_args(cls, args: argparse.Namespace) -> RunConfig:
             scheduler = None
 
         path = args.path
-        if (scale_factor := args.scale) is None:
+        name = Path(sys.argv[0]).name
```

Comment:
I see that this is just moving existing code, but this code path is reachable outside of running the CLI so relying on it for correctness is a bit fragile.

A way to do this might be to add the benchmark type / name (`pdsh` vs `pdsds`) as another argument to `from_args`, and have the caller (the pdsh or pdsds CLI, usually) pass it through.